src/main/java/main/view/BaseServlet.java

```diff
@@ -191,6 +193,15 @@ protected void processResponse(HttpServletResponse response, String filePath) {
         }
     }
 
+    protected void setResponseBody(HttpServletResponse resp, Object object) {
+        try {
+            setJSONContentType(resp);
+            resp.getWriter().write(mapper.serialize(object));
+        } catch (IOException e) {
+            System.out.println("Cannot set response body!");
```

Comment:
Is it okay to use System.out.println in catch? 
Could we use handleException method here or  what if we set e.g. 204 code?